db/schema.rb

```diff
@@ -54,20 +53,20 @@
     t.string "outcome"
     t.integer "processed_users", default: 0, null: false
     t.integer "total_users", default: 0, null: false
-    t.datetime "created_at"
-    t.datetime "updated_at"
+    t.datetime "created_at", precision: nil
+    t.datetime "updated_at", precision: nil
   end
 
   create_table "event_logs", id: :integer, charset: "utf8mb3", force: :cascade do |t|
     t.string "uid"
-    t.datetime "created_at", null: false
+    t.datetime "created_at", precision: nil, null: false
     t.integer "initiator_id"
     t.integer "application_id"
     t.string "trailing_message"
     t.integer "event_id"
     t.decimal "ip_address", precision: 38
     t.integer "user_agent_id"
-    t.string "user_agent_string"
+    t.text "user_agent_string"
```

Comment:
Oops, this change shouldn't have been in this commit.

It should've been included in https://github.com/alphagov/signon/pull/1843.